lms/djangoapps/courseware/views.py

```diff
@@ -982,7 +982,7 @@ def _progress(request, course_key, student_id):
             if certs_api.get_active_web_certificate(course) is not None:
```

Comment:
See below comment about keeping this call where it is -- similar situation in this spot.